spec/factories.rb

```diff
@@ -25,15 +24,68 @@
     sequence(:description) { |n| "Description of Event #{n}." }
     start_time { today + 1.hour }
     end_time { start_time + 1.hour }
-
     after(:create) { Sunspot.commit if Event::SearchEngine.kind == :sunspot }
+
+    trait :with_venue do
+      association :venue
+    end
   end
 
-  factory :event_with_venue, :parent => :event do
-    association :venue
+  factory :duplicate_event, parent: :event do
+    association :duplicate_of, factory: :event
   end
 
-  factory :duplicate_event, :parent => :event do
-    association :duplicate_of, :factory => :event
+  factory :seed_venue, parent: :venue do
+    title           { Faker::Company.name }
+    description     { Faker::Lorem.paragraph }
+    address         { "#{Faker::Address.street_address},
+                       #{Faker::Address.city},
+                       #{Faker::Address.state}
+                       #{Faker::Address.zip_code}" }
+    street_address  { Faker::Address.street_address }
+    locality        { Faker::Address.city }
+    region          { Faker::Address.state }
+    postal_code     { [Faker::Address.zip_code, Faker::Address.postcode].sample }
+    country         { Faker::Address.country }
+    latitude        { Faker::Address.latitude }
+    longitude       { Faker::Address.longitude }
+    email           { Faker::Internet.email }
+    telephone       { Faker::PhoneNumber.phone_number }
+    url             { Faker::Internet.url }
+    closed          { [false, true].sample }
+    wifi            { [true, false].sample }
+    access_notes    Faker::Lorem.paragraph
```

Comment:
Not necessarily — it's probably okay for all of the venues to have the same lorem ipsum access notes.